### PR TITLE
SIT course catalogue support

### PR DESCRIPTION
**Pull request by horrible-little-slime**
Drafted because it is untested rn; not cutting a new eslint-plugin-libram until after the new path, I think

### CODE REVIEW

**Comment by docrostov** [comment]
image
tested, it works fine it appears

File: src/tasks/dailyItems.ts [hunk 1]
```diff
@@ -302,5 +302,15 @@ export const DailyItemTasks: Task[] = [
       completed: () => get("_garbageItemChanged"),
       do: () => cliExecute("fold broken champagne bottle"),
     },
+    {
+      name: "Learn About Bugs",
+      // eslint-disable-next-line libram/verify-constants
+      ready: () => have($item`S.I.T. Course Completion Certificate`),
+      // eslint-disable-next-line libram/verify-constants
+      completed: () => get("_sitCourseCompleted", true) || have($skill`Insectologist`),
```
Comment (docrostov) [inline src/tasks/dailyItems.ts:310]: my only small note on this: if prefs get misaligned, and the user doesn't have insectologist but mafia thinks the sit course isn't completed, will this construction lead to an infinite loop?

**Review by docrostov — APPROVED** [review]
this works under my testing scenario of "not having insectology but being able to use the thing again." i do not see any reason why this wouldn't work under other scenarios; the only small question i have is when in the login process mafia checks skills. this feels like this should work pretty well tho!

File: src/tasks/dailyItems.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/tasks/dailyItems.ts:310]: a quick search in the mafia git repository tells me that `indelible-for-a-day` appears here:
https://github.com/kolmafia/kolmafia/blob/881e876fc2f41706259097fa9e41ab34b4779fcc/src/net/sourceforge/kolmafia/request/UseItemRequest.java#L6008-L6012

TL;DR: we're fine
